background.js

```diff
@@ -4,7 +4,7 @@
 browser.browserAction.onClicked.addListener(runLightbeam);
 
 async function runLightbeam() {
-
+  captureData();
```

Comment:
What a polite way of giving feedback! I have considered it and moved it outside of `runLightbeam`.